packages/api/src/controllers/stream.ts

```diff
@@ -896,18 +896,17 @@ app.post(
     // compatibility with /setactive recordging/webhooks handling)
     const issues =
       payload.is_active && !payload.is_healthy
-        ? payload.human_issues ||
-          (payload.issues ? [payload.issues] : undefined)
-        : undefined;
+        ? payload.human_issues || (payload.issues ? [payload.issues] : [])
+        : null;
     const patch: Partial<DBSession & DBStream> = {
-      isHealthy: payload.is_active ? payload.is_healthy : undefined,
+      isHealthy: payload.is_active ? payload.is_healthy : null,
       issues,
       // do not clear the `lastSeen` field when the stream is not active
       ...(payload.is_active ? { lastSeen: Date.now() } : null),
     };
 
     // Since we might need to delete some fields, use replace instead of update
```

Comment:
I think this comment can be removed

Reply:
Removed in bd9959fa85ca4c1f86d1a638960c4e7e226cc161